### DISCUSSION TITLE
Adding to awesome Go

### DISCUSSION THREAD

**Discussion by mohamedallam1991** [VinGarcia/ksql#28] [Show and tell]
I like this project so much, and I just sent a pull request to Awesome Go, to include it, please refer to the link below.
And approve the badges, to get accepted.

[aweome-go pull request](https://github.com/avelino/awesome-go/pull/4371)

**Comment by VinGarcia** [comment]
I am happy you liked it =], and thanks for all three contributions, I've just merged both of your PRs.

**Reply by mohamedallam1991** [reply]
The pleasure is all mines. (first FOSS contribution ;) )

**Reply by VinGarcia** [reply]
Nice =]

You are always welcome to contribute here =]

Also, I will share this other project with you because I am somewhat exciting with how it is turning out:

https://github.com/VinGarcia/structscanner

Maybe you will like this one too, regards.

**Reply by mohamedallam1991** [reply]
This seems like magic, I like this one too, I will try to use it in my learning. Thank you for sharing. :) 

After code coverage badge gets add and all, and the other package get approved, I would send a pull request to add this one as well, so the Go community gives it more love.

**Reply by VinGarcia** [reply]
It will be great, I am working on setting up the CI, the code coverage is probably very good since I am writing it with TDD from start.

**Reply by VinGarcia** [reply]
Done, we have the badges now for that project too.

**Reply by mohamedallam1991** [reply]
I saw you have TDD everything, awesome job 👏 , 
I just checked out the badges 🌟 , 
It looks fancier 🤖
I will work on getting it added to the curated Go list too! 🚀

**Reply by VinGarcia** [reply]
Thanks, I think I will have to add semver to it as well, I will do it right now.

**Reply by VinGarcia** [reply]
Done =]

**Reply by VinGarcia** [reply]
Actually you know what. It might be better to wait, this project is too green for now.

If we publish it there I won't be able to make some changes here and there, like renaming it from structsscanner to sscanner and I would even be afraid of making breaking changes to the API. 

It might be worth waiting for a while before actively promoting this one 🤔
